cmd/main.go

```diff
@@ -188,3 +191,20 @@ func getProjectVersion() (bool, string) {
 	}
 	return true, projectInfo.Version
 }
+
+func printV1DeprecationWarning() {
+	fmt.Printf(`=============== DEPRECATION WARNING ===============
+v1 projects are deprecated. The support for v1 projects will be dropped at 1 February 2020. 
```

Comment:
I dropped an approximate date, please correct me if it is wrong.

Reply:
What about a broad date such as "first 2020 quarter"? @droot

Reply:
Giving a specific date makes the message more clear and that's the intent behind this.